app/controllers/admin/loans_controller.rb

```diff
@@ -127,6 +127,7 @@ def print
     @mode = params[:mode]
     @first_image = @loan.media.find {|item| item.kind == 'image'}
     prep_attached_links if @mode != "details-only"
+    @breakeven_data = breakeven_data_stub
```

Comment:
@linesarefuzzy Not sure what is going on here. I thought breakeven was going to be a question type. I see this is for stubbing. But is that still the plan? Couldn't there thus be multiple tables in one loan memo?